Hippo.Web/Controllers/AccountController.cs

```diff
@@ -49,15 +51,23 @@ public async Task<ActionResult> Approve(int id)
     {
         var currentUser = await _userService.GetCurrentUser();
 
-        var account = await _dbContext.Accounts.SingleOrDefaultAsync(a => a.Id == id && a.Sponsor.OwnerId == currentUser.Id && a.Status == Account.Statuses.PendingApproval);
+        var account = await _dbContext.Accounts.Include(a => a.Owner).AsSingleQuery()
+            .SingleOrDefaultAsync(a => a.Id == id && a.Sponsor.OwnerId == currentUser.Id && a.Status == Account.Statuses.PendingApproval);
 
         if (account == null)
         {
             return NotFound();
         }
 
+        Console.WriteLine($"Approving account {account.Owner.Iam} with ssh key {account.SshKey}");
+
+        _sshService.PlaceFile(account.SshKey, $"/var/lib/remote-api/{account.Owner.Kerberos}.txt");
```

Comment:
@srkirkland What about using the IAM id?

Reply:
they want kerbID.  BUT they want it "in" the file so the name will probably change anyway